docs/design/trace_points/runtime_trace_points.md

```diff
@@ -137,6 +146,24 @@ Sampled items
 - uint64_t source_timestamp
 - uint64_t message_timestamp
 
+This tracepoint is no longer used from v0.4.10 onwards.
+
+---
+
+#### ros2:rmw_take (after v0.4.10)
+
+[Built-in tracepoints]
+
+Sampled items
+
+- void \* rmw_subscription_handle
+- void \* message
+- int64_t \* source_timestamp
+- bool \* taken
+
+In CARET, This tracepoint is used to correctly link the `callback_start` to the `rclcpp_publish` that triggered the callback..
```

Comment:
```suggestion
In CARET, This tracepoint is used to correctly link the `callback_start` to the `rclcpp_publish` that triggered the callback.
```

Reply:
Thank you for pointing this out.
It is my mistake.